src/main/blackbox/blackbox.c

```diff
@@ -1757,13 +1757,16 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_RC_SMOOTHING_AUTO_FACTOR, "%d",          rxConfig()->rc_smoothing_auto_factor_rpy);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_RC_SMOOTHING_AUTO_FACTOR_THROTTLE, "%d", rxConfig()->rc_smoothing_auto_factor_throttle);
 
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_RC_SMOOTHING_FEEDFORWARD_CUTOFF, "%d", rcSmoothingData->setpointCutoffSetting);
```

Comment:
Is this necessary? BB parser should handle missing header, it should not break it completely.

Reply:
Yeah, I don't like keeping this around when rc smoothing feedforward cutoff no longer exists.